app/src/componentsV2/BreadCrumb/BreadCrumb.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import "./breadcrumb.scss";
+
+interface BreadcrumbProps {
+  items: string[];
+}
+
+export const BreadCrumb: React.FC<BreadcrumbProps> = ({ items }) => {
+  return (
+    <div className="rq-breadcrumb">
+      {items.map((item, index) => (
+        <React.Fragment key={index}>
+          <span className="rq-breadcrumb-item">{item}</span>
+          {index !== items.length - 1 && " > "}
+        </React.Fragment>
+      ))}
+    </div>
+  );
+};
```

Comment:
Don't make a common component right now. Breadcrumb logic should be determined by routing and we'll be making them clickable too in future. So this won't scale much. 

Lets duplicate it everywhere for now